pom.xml

```diff
@@ -33,7 +33,7 @@
     <slf4j-api.version>1.7.3</slf4j-api.version>
     <commons-codec.version>1.9</commons-codec.version>
     <aspectjrt.version>1.6.6</aspectjrt.version>
-    <pentaho-reporting-engine-classic-core.version>8.1-SNAPSHOT</pentaho-reporting-engine-classic-core.version>
+    <pentaho-reporting.version>8.1.0.0-SNAPSHOT</pentaho-reporting.version>
```

Comment:
@smaring we already have pentaho-reporting.version defined on line 29, but I agree with this simplified approach